src/utils/format_date_util.js

```diff
@@ -4,3 +4,10 @@ export function formatDate(text) {
   var day = text.slice(8, 10);
   return month + '/' + day + '/' + year;
 }
+
```

Comment:
Here is the util we added to format the date data in a YYYY-MM-DD format

Reply:
LOVE the addition of a _reusable_ util for formatting dates - something that could occur frequently across the app.